splunk_eventgen/lib/plugins/generator/jinja.py

```diff
@@ -193,8 +193,7 @@ def gen(self, count, earliest, latest, samplename=None):
             if self._sample.splunkEmbedded is True:
                 splunk_home = os.environ["SPLUNK_HOME"]
                 app_name = getattr(self._sample, 'app', 'SA-Eventgen')
-                sample_dir = os.path.join(splunk_home, 'etc', 'apps', app_name,
-                                          'default', self._sample.DEFAULT_SAMPLE_DIR)
+                sample_dir = os.path.join(splunk_home, 'etc', 'apps', app_name, 'samples')
```

Comment:
Fix bug introduced in this pull request: https://github.com/splunk/eventgen/pull/177